Makefile

```
@@ -649,6 +654,15 @@ webhookserver-image: webhookserver-build
 		-t quay.io/rhacs-eng/webhookserver:1.2 \
 		-f webhookserver/Dockerfile webhookserver
 
+syslog-image: syslog-build
+	-mkdir syslog/bin
+	cp bin/linux_$(GOARCH)/syslog syslog/bin/syslog
+	chmod +w syslog/bin/syslog
+	docker build \
+		-t stackrox/syslog:1.0 \
+		-t quay.io/rhacs-eng/syslog:1.0 \
```

```suggestion
		-t quay.io/rhacs-eng/qa:syslog_server_1_0 \
```

We don't have a repository called `syslog` and instead of creating one I think we should use the `qa` one which is where all testing related stuff should go into. And we can use the tag to say this is syslog.